src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -1842,6 +1841,56 @@ class CreateStatementSegment(BaseSegment):
     )
 
 
+@snowflake_dialect.segment(replace=True)
+class CreateViewStatementSegment(BaseSegment):
+    """A `CREATE VIEW` statement, specifically for Snowflake's dialect.
+
+    https://docs.snowflake.com/en/sql-reference/sql/create-view.html
+    """
+
+    type = "create_view_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        Ref("OrReplaceGrammar", optional=True),
+        AnyNumberOf(
+            "SECURE",
+            "RECURSIVE",
+        ),
+        "VIEW",
+        Ref("IfNotExistsGrammar", optional=True),
+        Ref("ObjectReferenceSegment"),
```

Comment:
This is probably a holdover of the fact that there were multiple possible objects that could be being referenced in the generic CREATE statement.

In the rest of the codebase we tend to use `Ref("TableReferenceSegment")` to refer to view objects (since they are treated the same as tables in most rules).

Reply:
Updated: https://github.com/sqlfluff/sqlfluff/pull/2217/files#diff-a8da7f7dc8f0e448950f6d5c2cd6cd5590026ea6da32cdc8f233e037bea947ecR1861